### DISCUSSION TITLE
Program Panic attach console output

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1252] [General]
It is random and it happen in past also but I don't know what I was doing 

```
thread '<unnamed>' panicked at src/lib.rs:3974:36:
invalid texture
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
SIGABRT: abort
PC=0x7fd9a36a949c m=7 sigcode=18446744073709551610
signal arrived during cgo execution

```
[panic.log](https://github.com/user-attachments/files/17329766/panic.log)

**Comment by kkoreilly** [comment]
Is it possible it happened when you resized the window, similar to #1247?

**Comment by heejit** [comment]
I can't remember exactly what I was doing that time.

I will try if I can find when it is happening.
